### PR TITLE
Bump `tokio` to 1.17.0

### PR DESCRIPTION
**Pull request by koute**
polkadot companion: https://github.com/paritytech/polkadot/pull/4965

cumulus companion: https://github.com/paritytech/cumulus/pull/1014

The newest version of `tokio` apparently has some potentially significant performance improvements. ([changelog here](https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md))

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by ggwpez** [comment]
Looks like you also need to update it for Polkadot since the versions are incompatible?

**Comment by wigy-opensource-developer** [comment]
Just mentioning here that libp2p still uses tokio 1.15

**Comment by bkchr** [comment]
> Just mentioning here that libp2p still uses tokio 1.15

This is not true? `1.17.0` is used everywhere in Substrate. `1.15.0` is also semver compatible to `1.17.0`.

**Comment by wigy-opensource-developer** [comment]
Yeah, right. I think I still cached 0.17 that was in the issue title in my head, sorry. :smiling_face_with_tear:

**Comment by koute** [comment]
Oops, sorry, yeah, I brainfarted and put `0.17.0` in the commit message since I'm so used to everything in the ecosystem being on the 0th version. (:

**Comment by KiChjang** [comment]
It now seems to also need a PR to cumulus as well, bumping `tokio` to version `1.17`.

**Review by wigy-opensource-developer — APPROVED** [review]

File: utils/frame/rpc/support/Cargo.toml [hunk 1]
```diff
@@ -25,5 +25,5 @@ sc-rpc-api = { version = "0.10.0-dev", path = "../../../../client/rpc-api" }
 
 [dev-dependencies]
 frame-system = { version = "4.0.0-dev", path = "../../../../frame/system" }
-scale-info = "2.0.0"
-tokio = "1.15"
+scale-info = "1.0"
```
Comment (KiChjang) [inline utils/frame/rpc/support/Cargo.toml:28]:
```suggestion
scale-info = "2.0.0"
```

**Comment by bkchr** [comment]
@koute this requires some master merge

**Pull request merged**